cmd/kubeseal/main.go

```diff
@@ -59,6 +61,8 @@ var (
 	fromFile       = flag.StringSlice("from-file", nil, "(only with --raw) Secret items can be sourced from files. Pro-tip: you can use /dev/stdin to read pipe input. This flag tries to follow the same syntax as in kubectl")
 	sealingScope   ssv1alpha1.SealingScope
 	reEncrypt      bool // re-encrypt command
+	unseal         = flag.Bool("recovery-unseal", false, "Decrypt a sealed secrets file obtained from stdin, using the private key passed with --recovery-private-key. Intended to be used in disaster recovery mode.")
+	privKeys       = stringlist.Flag("recovery-private-key", "Private key filename used by the --recovery-unseal command. Multiple files accepted either via comma separated list of by repetition of the flag.")
```

Comment:
Is this the same as `flag.StringSlice`?  https://godoc.org/github.com/spf13/pflag#StringSlice

Reply:
Uh, I forgot we were using pflag here...